example/unity/DemoApp/Assets/FlutterUnityIntegration/Editor/Build.cs

```diff
@@ -189,7 +203,17 @@ private static void DoBuildAndroid(String buildPath, bool isPlugin)
             playerOptions.scenes = GetEnabledScenes();
             playerOptions.target = BuildTarget.Android;
             playerOptions.locationPathName = APKPath;
-            playerOptions.options = BuildOptions.AllowDebugging;
+            if (!isReleaseBuild)
+            {
+                playerOptions.options = BuildOptions.AllowDebugging;
+            }
+            #if UNITY_2022_1_OR_NEWER
+				PlayerSettings.SetIl2CppCompilerConfiguration(BuildTargetGroup.Android, Il2CppCompilerConfiguration.Debug);
```

Comment:
@Ahmadre 
Shouldn't these player settings be affected by `isReleaseBuild` too?

Reply:
Thank you @timbotimbo ! You're totally right. Will add a check here.

Reply:
@timbotimbo Done. Can we do that also for iOS Target? Or does that lead to errors?

Reply:
I missed the other setting on the next line in my code quote. `Il2CppCodeGeneration.OptimizeSize` might worsen performance in release builds too.

I haven't tested these settings on iOS yet. In theory they shouldn't break anything, just make building a little faster and reduce runtime performance a little. But its always a good idea to test it first.

Reply:
Alright. Then I will test it now with a production app on my side. 

Actually it's only iOS. So If it doesn't change significally I will leave iOS out.

Reply:
Ok, I tested it and it's very fast on my iOS device! I really feel the "Release" Build :D